dspace/config/spring/api/metadata-enhancers.xml

```diff
@@ -57,6 +57,34 @@
         <property name="relatedItemMetadataField" value="person.identifier.orcid" />
         <property name="virtualQualifier" value="editor-orcid" />
     </bean>
+
+    <!--  Set dc.title of person from some of their other name fields -->
+    <bean class="org.dspace.content.enhancer.impl.NameEnhancer" >
+        <property name="sourceEntityType" value="Person" />
+        <property name="targetItemMetadataField" value="dc.title" />
+        <property name="relatedItemMetadataFields">
+        	<list>
+        		<value>crisrp.preferredName</value>
+        		<value>crisrp.fullName</value>
+        	</list>
+        </property>
+        <property name="defaultValue" value="N.N." />
+    </bean>
+
+<!-- enhance values using some mapConverter -->
+    <bean class="org.dspace.content.enhancer.impl.MapConverterValueEnhancer" >
+        <property name="sourceEntityType" value="Publication" />
+        <property name="sourceItemMetadataField" value="dc.type" />
+        <property name="targetItemMetadataField" value="dc.type.test" />
+        <property name="converter" ref="mapConverterDSpaceToCoarTypes" />
+        <property name="useDefaultLanguage" value="true" />
+    </bean>
+<!-- bean from crosswalks.xml not found. if some error occurs rename it -->
+    <bean name="mapConverterDSpaceToCoarTypes" class="org.dspace.util.SimpleMapConverter" init-method="init">
+        <property name="converterNameFile" value="mapConverter-dspace-to-coarTypes.properties" />
```

Comment:
This file `mapConverter-dspace-to-coarTypes.properties` is not contained in this PR. Could you please provide a sample file with the expected format.

Reply:
changed the file :) because dspace7 uses some controlled vocabulary, https://github.com/4Science/DSpace/blob/dspace-cris-7/dspace/config/crosswalks/mapConverter-coarTypes.properties should be used as input for the converter.